scripts/testnet/include/config.sh

```diff
@@ -8,9 +8,9 @@ generateConfig() {
     TMP_META_OBSERVERCOUNT=0
   fi
 
-  SOVEREIGN_BOOL="false"
+  SOVEREIGN_DEPLOY=0
   if [ $USE_HARDFORK -eq 1 ]; then
-    SOVEREIGN_BOOL="true"
+    SOVEREIGN_DEPLOY=1
```

Comment:
Wrong

```
  SOVEREIGN_BOOL="false"
  if [ $SOVEREIGN_DEPLOY -eq 1 ]; then
    SOVEREIGN_BOOL="true"
```

SOVEREIGN_DEPLOY -> search name of this variable exists  and is 1/0 to make sure